spec/arbre/unit/html/tag_spec.rb

```diff
@@ -3,33 +3,35 @@
 
 describe Arbre::HTML::Tag do
 
-  let(:tag){ Arbre::HTML::Tag.new }
+  let(:tag){ described_class.new }
 
   describe "building a new tag" do
     before { tag.build "Hello World", id: "my_id" }
 
-    it "should set the contents to a string" do
+    it "sets the contents to a string" do
       expect(tag.content).to eq("Hello World")
     end
 
-    it "should set the hash of options to the attributes" do
+    it "sets the hash of options to the attributes" do
       expect(tag.attributes).to eq({ id: "my_id" })
     end
   end
 
+  # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
I believe this comment and the enable one, are only needed around the `let` calls where the `double` is used. It shouldn't be needed around the entire `describe` block.

Ah I see there is another nested on line 40. I would recommend wrapping these directly with the comment to keep them as local as possible.